python/cudf/cudf/core/column/categorical.py

```diff
@@ -956,9 +956,8 @@ def clip(self, lo: ScalarLike, hi: ScalarLike) -> "column.ColumnBase":
             self.astype(self.categories.dtype).clip(lo, hi).astype(self.dtype)
         )
 
-    @property
-    def data_array_view(self) -> cuda.devicearray.DeviceNDArray:
-        return self.codes.data_array_view
+    def data_array_view(self, mode="read") -> cuda.devicearray.DeviceNDArray:
```

Comment:
Can we default to safe but slow `"write"`? Or does `"write"` not differ from `read` for this case?

Reply:
Still `"read"` here AFAICT?

Reply:
Sorry forgot to do so. Thanks for catching it. Should be fixed now.